cg/models/report/metadata.py

```diff
@@ -87,16 +87,13 @@ class BalsamicWGSSampleMetadataModel(BalsamicSampleMetadataModel):
     pct_reads_improper_pairs: Annotated[str, BeforeValidator(get_float_as_string)] = NA_FIELD
 
 
-class RnafusionSampleMetadataModel(SampleMetadataModel):
-    """Metrics and trending data model associated to a specific Rnafusion sample.
+class NextflowSampleMetadataModel(SampleMetadataModel):
```

Comment:
I would call this "common" or "common_wgs" or "common_wts" o something similar,  since it doesn't have anything nothing to do with nextflow. This will also live very shortly as taxprofiler won't have most of these fields. So maybe it it can be skipped

Reply:
Sounds good 👍 

Changed it to `WTSSampleMetadataModel`